src/galileo/resources/api/auth/login_api_key_login_api_key_post.py

```diff
@@ -16,6 +16,7 @@
 
 def _get_kwargs(*, body: ApiKeyLoginRequest) -> dict[str, Any]:
     headers: dict[str, Any] = {}
+    headers: dict[str, Any] = {}
```

Comment:
Looks like the `headers` declaration is duplicated in a number of files.. 

Guess it doesn't matter but not sure if it's due to an issue in the template or not